### DISCUSSION TITLE
MFA报错

### DISCUSSION THREAD

**Discussion by yunqi777** [PaddlePaddle/PaddleSpeech#2598] [Q&A] [answered]
我抽取了数据集中的一部分进行mfa却成功了
![image](https://user-images.githubusercontent.com/110005064/198856563-099c5b10-4d3b-4ae7-b8f1-8bfb8a1887cb.png)

**Comment by yunqi777** [comment]
如果全部mfa就会报错

**Comment by yunqi777** [comment]
wav和txt对应文件能够生成，但是对齐结果就是出不来
![image](https://user-images.githubusercontent.com/110005064/198856634-ed92186e-c708-4516-bab0-4f720da43b0c.png)

**Comment by yunqi777** [comment]
换了128G显存还是不够。。。
![image](https://user-images.githubusercontent.com/110005064/198863085-49f718ec-7b7d-4bd7-8570-a398dfd3abe9.png)

**Comment by yt605155624** [comment, marked as answer]
如报错所示，输入音频的采样率需要保持一致，请采样到一致的 sample rate 后再进行 MFA
